src/common/transformations/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ ie_faster_build(${TARGET_NAME}_obj
     PCH PRIVATE "src/precomp.hpp"
 )
 
-target_link_libraries(${TARGET_NAME}_obj PRIVATE ngraph_reference openvino::itt ngraph::builder openvino::core::dev)
+target_link_libraries(${TARGET_NAME}_obj PRIVATE ngraph_reference openvino::itt ngraph::builder openvino::core::dev ov_shape_inference)
```

Comment:
This change is to include `augru_sequence_shape_inference.hpp` in 
`src/common/transformations/src/ngraph_ops/augru_sequence.cpp`